### ISSUE TITLE
Scale values on objects/bones seem to not be handled well and cause issues

### ISSUE THREAD

**Issue by wojtossfm** [wojtossfm/godot_scml_importer#7]
Project with issue submitted on https://github.com/wojtossfm/godot_scml_importer/pull/5#commitcomment-46733784

Blocked by #6

**Comment by fabiodellaria** [comment]
Any news?

**Comment by wojtossfm** [comment]
No definitive fixes yet I'm afraid. I've found a change that is a partial fix but haven't yet found a solution that makes it look correct. Feel free to experiment if you have time.

**Comment by wojtossfm** [comment]
So I believe I found a solution that works for the scaling issues. Found another issue with the animations expecting to reparent object between different animations which isn't currently supported - see #10
